fe/fe-core/src/main/java/org/apache/doris/statistics/HMSAnalysisTask.java

```diff
@@ -65,7 +65,7 @@ public class HMSAnalysisTask extends BaseAnalysisTask {
             + "MAX(`${colName}`) AS max, "
             + "${dataSizeFunction} AS data_size, "
             + "NOW() "
-            + "FROM `${catalogName}`.`${dbName}`.`${tblName}` ${sampleExpr}";
+            + "FROM `${catalogName}`.`${dbName}`.`${tblName}`";
```

Comment:
it seems that you changed grammar, why there is no change on .g4 files?

Reply:
No, we didn't change any grammar